chrono include is interfering in compiling with C API in header xrt_graph.h

```c
#include <chrono>
```
in header XRT/src/runtime_src/core/include/experimental/xrt_graph.h interferes in compiling with C API.

```bash
/experimental/xrt_graph.h:22:10: fatal error: chrono: No such file or directory
   22 | #include <chrono>
      |          ^~~~~~~~
compilation terminated.
make: *** [main.o] Error 1

```

--- a/src/runtime_src/core/include/experimental/xrt_aie.h
+++ b/src/runtime_src/core/include/experimental/xrt_aie.h
@@ -26,6 +26,11 @@
 #include "experimental/xrt_graph.h"
 
 #ifdef __cplusplus
+# include <cstdint>
+# include <string>
+#endif
+
+#ifdef __cplusplus
 
 namespace xrt { namespace aie {
 

--- a/src/runtime_src/core/include/experimental/xrt_error.h
+++ b/src/runtime_src/core/include/experimental/xrt_error.h
@@ -22,8 +22,12 @@
 #include "xrt_device.h"
 
 #ifdef __cplusplus
-#include <string>
+# include <string>
+# include <memory>
+# include <cstdint>
+#endif
 
+#ifdef __cplusplus
 namespace xrt {
 
 class error_impl;
@@ -98,7 +102,7 @@ private:
  */
 XCL_DRIVER_DLLESPEC
 int
-xrtErrorGetLast(xrtDeviceHandle handle, xrtErrorClass ecl, xrtErrorCode* error, uint64_t* timestamp);
+xrtErrorGetLast(xrtDeviceHandle handle, enum xrtErrorClass ecl, xrtErrorCode* error, uint64_t* timestamp);
 
 /**
  * xrtErrorGetString - Get the description string of a given error code.

--- a/src/runtime_src/core/include/experimental/xrt_graph.h
+++ b/src/runtime_src/core/include/experimental/xrt_graph.h
@@ -19,12 +19,16 @@
 #ifndef XRT_GRAPH_H_
 #define XRT_GRAPH_H_
 
-#include <chrono>
-
 #include "xrt.h"
 #include "xrt/xrt_uuid.h"
 #include "xrt/xrt_bo.h"
 #include "xrt/xrt_device.h"
+
+#ifdef __cplusplus
+# include <chrono>
+# include <string>
+# include <cstdint>
+#endif
 
 typedef void *xrtGraphHandle;
 

--- a/src/runtime_src/core/include/xrt_error_code.h
+++ b/src/runtime_src/core/include/xrt_error_code.h
@@ -40,6 +40,8 @@
 
 #if defined(__KERNEL__)
 # include <linux/types.h>
+#elif defined(__cplusplus)
+# include <cstdint>
 #else
 # include <stdint.h>
 #endif

